src/FSharpLint.Core/DefaultConfiguration.FSharpLint

```diff
@@ -327,6 +327,11 @@
           <OneSpaceAllowedAfterOperator>true</OneSpaceAllowedAfterOperator>
           <IgnoreBlankLines>true</IgnoreBlankLines>
         </TrailingWhitespaceOnLine>
+
+        <Indentation>
+          <Enabled>true</Enabled>
```

Comment:
I think we should either have this disabled by default or try to auto detect their indentation, otherwise anyone with different valid indentation might have to start adding lint config files (or more likely disable the linter in their editor)

Reply:
Agreed.